webServer.js

```diff
@@ -255,6 +257,15 @@ function start() {
     });
   }
 
+  function libraryMetaInfo(library) {
+    if (library.repository !== undefined) {
```

Comment:
I'll suggest you to directly drop those non-GitHub repo here since it's a simply match, there is no need to do an additional parse and string handling to waste the resource.